packages/fuels-test-helpers/src/utility.rs

```diff
@@ -0,0 +1,103 @@
+use std::future::Future;
+use std::time::Duration;
+
+pub async fn retry_until<Fut>(
+    condition: impl Fn() -> Fut,
+    max_attempts: usize,
+    between_attempts: Duration,
```

Comment:
Call this argument `interval`, this name is a shortened version of `time_between_attempts`, which would be a good name, but too long, but its shortened version isn't clear enough.